src/lib/core/CHIPPersistentStorageDelegate.h

```diff
@@ -46,30 +46,22 @@ class DLL_EXPORT PersistentStorageDelegate
      *   Caller is responsible to take care of any special formatting needs (e.g. byte
      *   order, null terminators, consistency checks or versioning).
      *
-     *   This API allows for determining the size of a stored value. Whenever
-     *   the passed `size` is smaller than needed and the key exists in storage, the error
-     *   CHIP_ERROR_BUFFER_TOO_SMALL will be given, and the `size` will be updated to the
-     *   size of the stored value. It is legal to use `nullptr` for `buffer` if `size` is 0.
-     *
      *   If a key is found and the `buffer`'s `size` is large enough, then the value will
      *   be copied to `buffer` and `size` will be updated to the actual size used.
      *
-     *   The easiest way to determine if a key exists (and the value's size if so) is to pass
-     *   `size` of 0, which is always valid to do, and will return CHIP_ERROR_BUFFER_TOO_SMALL
-     *   if the key exists and CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND if the
-     *   key is not found.
+     *   Whenever the passed `size` is smaller than the value for which the key exists in storage,
```

Comment:
Why did we delete this text that mentions the use of size = 0 to infer the presence/absence of a key by looking for the `CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND` vs `CHIP_ERROR_BUFFER_TOO_SMALL` error?

Reply:
Ah - is this removal to bias people towards the `SyncDoesKeyExist` API? If so, a little blurb here about using that instead might be worthwhile.

Reply:
Done. PTAL